flake8_quotes/__about__.py

```diff
@@ -1 +1 @@
-__version__ = '3.2.0'
+__version__ = '4.0.0'
```

Comment:
We can prob drop this change, I usually do the update via `./release.sh` which adds this change to the `git tag` commit for good measure/traceability

```suggestion
__version__ = '3.2.0'
```

Reply:
Cool. Personally I use a CI job to copy the git tag into the setup.py file and in code I grab the version from pkg_resources.